src/test/util/txmempool.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_TEST_UTIL_TXMEMPOOL_H
+#define BITCOIN_TEST_UTIL_TXMEMPOOL_H
+
+#include <txmempool.h>
+
+struct TxMemPoolClearable : public CTxMemPool {
```

Comment:
May I suggest to name this struct more generally, say `TxMemPoolTesting`, as it seems more functions could be added to it.
E.g., a special version of `GetTransactionAncestry()` that requires external `CTxMemPool::cs` locking, see https://github.com/bitcoin/bitcoin/pull/19872#discussion_r484841181.

Reply:
Ignore all above. It won't work.